streams/src/main/java/org/apache/kafka/streams/state/internals/CachingKeyValueStore.java

```diff
@@ -142,12 +140,12 @@ public void close() {
 
     @Override
     public boolean persistent() {
-        return underlying.persistent();
+        return super.persistent();
     }
 
     @Override
     public boolean isOpen() {
-        return underlying.isOpen();
+        return super.isOpen();
```

Comment:
Not introduced in this PR but: why do we still need to override these two functions? Should the ones of `WrappedStore` be sufficient?

Reply:
It was an oversight. I'll remove the pass-though overrides.